src/redux/actions/likeDislike.js

```diff
@@ -0,0 +1,90 @@
+import axios from "axios";
+import { toaster } from "evergreen-ui";
+import {
+  LIKE_ARTICLE,
+  DISLIKE_ARTICLE,
+  LIKE_DISLIKE_ARTICLE_ERROR
+} from "../action_types";
+import config from "../../config";
+
+const url = `${config.BASE_URL}/articles/`;
+
+/**
+ * Like an article
```

Comment:
I'm fairly certain this is not how I taught you all to write comments. Please consult the likes of @PAbishai and @meshack-mbuvi and refactor all these comments

Reply:
This has been fixed